Update readme.md (#106)
manuelalferez/flowinance: CONTRIBUTING.md

# Contributing to Flowinance

> We’re so glad you’re interested in contributing to our **open-source project**! Before getting started, please take a moment to read our guidelines to ensure a smooth and efficient contribution process.

Reviewer: ```suggestion
We are glad that you are interested in contributing to our open source project!
```

When contributing to this repository, please first discuss the change you wish to make [**creating an issue**](https://github.com/manuelalferez/flowinance/issues).
We are glad that you are interested in contributing to our open source project!

## Pull Request Process
## 📝 Contribution Guidelines
1. Discuss Changes Before Starting
Please first discuss the change you wish to make [**creating an issue**](https://github.com/manuelalferez/flowinance/issues).

1. Make sure the change runs successfully locally
2. Consider if the change affects `README.md`
3. In your commit message's subject (50 char length at max) you should succinctly describe what changes this commit introduces

> 4. A commit shouldn't take more than 100 lines of code. You will write your contribution faster and the person who will review your code will appreciate it.

Reviewer: Please, keep this.

> - Install all necessary dependencies and confirm the app runs as expected.
> 
> 3. Commit Messages
> - Use concise and clear commit messages that describe the change.
> - Limit your subject line to 50 characters max and succinctly describe what changes the commit introduces.
> 
> 
> 4. Code Quality & Best Practices
> - Keep your changes small and focused. Limit pull requests to around 100 lines of code to ease the review process.
> - Consider whether your change affects documentation (README.md, etc.).
> 
> <br>
> 
> ## 📑 Steps to Make Your First Contribution
> To help in contributing, you can follow this simple guidelines:
> 
> ### 1. Fork the repository.
> 
> ### 2. Clone your fork
> ```bash
> git clone https://github.com/[your-github-username]/flowinance.git
> ```
> 
> ### 3. Navigate to the directory
> ``` bash
> cd flowinance
> ```
> ### 4. Create a new branch for your feature/bugfix:
> ```bash
> git checkout -b your-branch-name
> ```
> 
> ### 5. Stage and Commit your changes:
> ```bash
> git add .
> ```
> 
> ```bash
> git commit -m "Add your message here"
> ```
> 
> ### 6. Push your changes to GitHub:
> ```bash
> git push origin your-branch-name
> ```
> 
> ### 7. Create a pull request and describe the changes.
>  Please refer to the [pull request format](https://github.com/manuelalferez/flowinance/blob/main/.github/PULL_REQUEST_TEMPLATE.md)
> 
>  <br>
> 
> ## 💬 Have feedback or questions? Join the conversation!
> 
> We encourage you to visit our [Flowinance Discussions](https://github.com/manuelalferez/flowinance/discussions) for general questions, feature suggestions, or to engage with the community. It's the best place to share ideas and collaborate!
> 
> ## Ask me 🤙
> 
> You can always contact me via [Telegram](https://t.me/manuelalferez) if you want to talk more about the project 😊
> 
> Thank you for your contributions! Together, we can make Flowinance even better 🎉

Reviewer: ```suggestion
```

Reviewer: We make contributors read a lot: we will respect everyone's time.

Author: @manuelalferez Apologies. I wanted to break it down for first-time contributors, I guess it isn't necessary. Apologies again
